assets/model_monitoring/components/model_monitor/model_monitor_data_joiner/spec.yaml

```diff
@@ -0,0 +1,57 @@
+$schema: http://azureml/sdk-2-0/SparkComponent.json
+type: spark
+
+name: model_monitor_data_joiner
+display_name: Model Monitor - Data Joiner
+description: Joins the data for model monitor.
```

Comment:
How about something a tad more specific, maybe
"Joins two data assets on a given join column for model monitor"

Reply:
Haha, yes sounds good.